dominion/frontend/src/component/Footer/Partners.js

```diff
@@ -1,15 +1,64 @@
 import React from 'react';
-import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
 
-function Partners() {
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+// Assets
+import bmf from '../../assets/images/logos/bmgf.png';
+import codeforafrica from '../../assets/images/logos/codeforafrica.png';
+import datazetu from '../../assets/images/logos/datazetu.png';
+import icfj from '../../assets/images/logos/icfj.png';
+import twaweza from '../../assets/images/logos/twaweza.png';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.secondary.light,
+    padding: theme.spacing.unit * 3
+  },
+  img: {
+    maxWidth: '100%',
+    height: 'auto'
+  },
+  imageGrid: { padding: theme.spacing.unit * 2 }
+});
+
+function Partners({ classes }) {
   return (
-    <Grid container direction="column" justify="center" alignItems="center">
-      <Typography variant="h5">Dominion Setup</Typography>
-      <Grid style={{ paddingTop: '2rem' }}>
-        <Typography variant="h6">Partner&#39;s Logos</Typography>
+    <Grid
+      container
+      spacing={24}
+      direction="row"
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item lg={2} md={6} sm={6} xs={6} style={{ padding: '2rem' }}>
```

Comment:
Arrange them from mobile -> desktop and not the other way around i.e Mobile first.